homeassistant/components/emoncms/sensor.py

```diff
@@ -198,22 +206,22 @@ def __init__(
             self._state = None
 
     @property
-    def name(self):
+    def name(self) -> str:
```

Comment:
Can you use shorthand attributes instead: `self._attr_name`?

Reply:
Done